Example/FusumaExample/Sources/FSAlbumView.swift

```diff
@@ -89,10 +89,10 @@ final class FSAlbumView: UIView, UICollectionViewDataSource, UICollectionViewDel
         if images.count > 0 {
             
             changeImage(images[0] as! PHAsset)
+            collectionView.reloadData()
+            collectionView.selectItemAtIndexPath(NSIndexPath(forRow: 0, inSection: 0), animated: false, scrollPosition: UICollectionViewScrollPosition.None)
```

Comment:
![image](https://cloud.githubusercontent.com/assets/6068413/14271334/941c2540-fb2a-11e5-82d9-c19bbd08f829.gif)
Once open gallery will select the first cell.